pmd-java/src/test/kotlin/net/sourceforge/pmd/lang/java/ast/OverrideResolutionTest.kt

```diff
@@ -37,24 +43,30 @@ class OverrideResolutionTest : ProcessorTestSpec({
                     super.foo(); // useless
                 }
             }
-        """)
-        val (fooFoo, barFoo, supFoo, subFoo) = acu.descendants(ASTMethodDeclaration::class.java).toList()
-        supFoo.overriddenMethod shouldBe barFoo.genericSignature
-        subFoo.overriddenMethod shouldBe fooFoo.genericSignature
-        barFoo.overriddenMethod shouldBe null
-        fooFoo.overriddenMethod shouldBe null
+        """
+            )
+            val (fooFoo, barFoo, supFoo, subFoo) = acu.descendants(ASTMethodDeclaration::class.java).toList()
+            supFoo.overriddenMethod shouldBe barFoo.genericSignature
+            subFoo.overriddenMethod shouldBe fooFoo.genericSignature
+            barFoo.overriddenMethod shouldBe null
+            fooFoo.overriddenMethod shouldBe null
+        }
     }
 
     parserTest("Test override resolution unresolved") {
-        val acu = parser.parse("""
+        doTest {
```

Comment:
Instead of needing to wrap everything in a doTest, could we redefine the method `parserTest(String)` to add a doTest internally, and add a new `parserTestContainer` method for tests that want to call doTest directly?

Reply:
I've added now a `parserTestContainer` and refactored all tests to use either `parserTest` or `parserTestContainer`.